### DISCUSSION TITLE
Why `0` is used in `&mt_home`, `&mt_end`, and `&smart_num`?

### DISCUSSION THREAD

**Discussion by numToStr** [urob/zmk-config#56] [Q&A] [answered]
I was looking at the keymap and saw that `0` is being used in `&smart_num` hold-tap modifier. But i couldn't understand why, does it mean something or is it just a placeholder due to bindings syntax?
https://github.com/urob/zmk-config/blob/de65ae61449216b43096ebd3b180404f4fa6ce2b/config/base.keymap#L112
https://github.com/urob/zmk-config/blob/de65ae61449216b43096ebd3b180404f4fa6ce2b/config/base.keymap#L84-L85

**Comment by urob** [comment, marked as answer]
TL;DR; It's a dummy parameter since we are only needing one parameter but the hold-tap expects two. The official docs go into a bit more details: https://zmk.dev/docs/behaviors/hold-tap#using-different-behavior-types-with-hold-taps
